svg-slideshow.py

```diff
@@ -29,6 +31,7 @@ def add_image(img):
 
 if __name__ == '__main__':
     parser = init_parser()
+    parser.print_help()
```

Comment:
This should probably be a fixup in the other PR? (but shouldn't it only be shown in the wrong use case or something like `--help`?)